src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -2013,43 +2011,84 @@ tools to limit workflow activity.
 Runahead Limiting
 ^^^^^^^^^^^^^^^^^
 
-Runahead limiting, controlled by :cylc:conf:`[scheduling]runahead limit`,
-prevents the fastest tasks in a workflow from getting too far
-ahead (with respect to cycle point) of the slowest ones.
+The scheduler runahead limit determines how many consecutive cycle points can
+be active at once. The base point of the runahead calculation is the current
+lowest-valued point with :term:`active` or :term:`incomplete` tasks present.
 
-.. note::
+The runahead limit can be set with :cylc:conf:`[scheduling]runahead limit`,
+to an integer interval (for datetime or integer cycling workflows) or to
+a datetime interval (datetime cycling only). 
 
-   Runahead limiting does not restrict activity within a cycle point.
```

Comment:
Is this note not worth keeping? I think it helps clarify what a runahead limit can and can't do. Especially now queues actually have a performance benefit?

Reply:
Yes it is - I just moved it, and highlighted it in an "note" box. See lines 2080-2084 below

Reply:
Sorry - missed that.